events/CloudFront/CreateFunction2020_05_31.json

```diff
@@ -9,6 +9,51 @@
     "mitreAttackTechniques": [
         "T1119 - Automated Collection"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1059 - Command and Scripting Interpreter",
+            "reason": "CloudFront functions are written in JavaScript, enabling the execution of scripts."
+        },
+        {
+            "technique": "T1106 - Native API",
+            "reason": "The creation of a CloudFront function involves direct interaction with AWS APIs, potentially leveraging native functionalities."
```

Comment:
You are right on that. My thought process was that inside AWS this is the native API. Maybe the framework needs to have something similar for cloud and container environments.

Reply:
I agree, I think there might be a lack of cloud-related techniques in MITRE framework but I think it is also because MITRE only has techniques that come from real-word observations, and cloud attacks are still "basic".  So as with most of my other comments, I'm not saying is bad but it might generate confusion based on MITRE techniques description and scope.

Reply:
I will remove it for now. If it is still needed we can bring it back.